### ISSUE TITLE
The comment windows doesn't show introduction

### ISSUE THREAD

**Issue by LAfricain** [crosswire/xiphos#1091]
In the mhc new release We have the introduction before every books. It's impossible to displays it.

**Comment by ariasrodolfo** [comment]
right-click in the commentary view pane and select -> Display book headings?

**Comment by ariasrodolfo** [comment]
I'll take advantage of this space to highlight a problem with the headings view. The -thesword- format allows when creating a module to assign various types of text as introduction,
1. Introduction of the complete module
2. Introduction of Each Testament
3. Introduction to a specific Book
4. Introduction to each specific chapter of a book
 
the view panel of the modules shows by default the text of each particular entry, and the introductory texts: 3. to a book and; 4. to a chapter, they are shown with the options: Display book heading, and Display chapter heading respectively, however Xiphos does not show the introduction to each testament, nor to the complete module, or at least I don't know how to access it.

**Comment by karlkleinpaste** [comment]
Yes, I know this is old.

I just tried out current MHC, and I agree, it doesn't seem to work, and commenting out the OSISHeadings qualifier makes it visible but of course then headings stop being toggle-able.

I'll take a look at some point.
